src/Game/Managers/NameOverHeadManager.cs

```diff
@@ -95,22 +95,28 @@ public static bool IsAllowed(Entity serial)
 
         public static void Open()
         {
-            if (_gump != null)
+            if (_gump == null)
             {
-                return;
+                _gump = new NameOverHeadHandlerGump();
             }
 
-            _gump = new NameOverHeadHandlerGump();
+            if (!_gump.IsEnabled)
+                _gump.IsEnabled = true;
+            if (!_gump.IsVisible)
+                _gump.IsVisible = true;
+
             UIManager.Add(_gump);
```

Comment:
the gump must be added when instanciated and not everytime the `Open` function is called.